src/plugins/inputs/who.js

```diff
@@ -0,0 +1,83 @@
+"use strict";
+const Msg = require("../../models/msg");
+
+exports.commands = ["who"];
+
+const parseWhoxResponse = (user, args) => {
+	const parseFieldNameForLounge = (field) => {
+		switch (field) {
+			case "channel":
+				return "channel"; // no-op for now
```

Comment:
This is the same thing that the default branch of this switch statement would do. What's the significance of having it separately called out here? Are you expecting it to not be a no-op in the future?

Reply:
I think I liked the explicitness at the time, but now 🤷